### DISCUSSION TITLE
how to get quarto project output-dir in R

### DISCUSSION THREAD

**Discussion by Liripo** [quarto-dev/quarto-cli#4155] [Q&A] [answered]
```bash
# example run command
quarto render --output-dir test
```
Is it possible to get the quarto command line parameter output-dir in R？

**Comment by mcanouil** [comment]
At this stage I would not recommend the `quarto` R package.
This being said, because Quarto CLI is a CLI, you can use `system()` and all the tools like that, see <https://ropensci.org/blog/2021/09/13/system-calls-r-package/>.

**Reply by cderv** [reply]
> At this stage I would not recommend the quarto R package.

Why is that ?

**Reply by mcanouil** [reply]
Last time I checked there was some issues with `renv` and you have a very limited access to Quarto CLI.

Also `quarto` was already behind at 1.0 release of Quarto CLI, but it's now way behind.

It should be "safe" to use the `quarto` R wrapper package as long as you only need to do `quarto render file.qmd` from R.

**Reply by cderv** [reply]
We'll make time to update the package soon but anything available through it should be working. Otherwise, opening an issue there is the way. 

That being said, using the **quarto** R package is useful to create some workflow from R. Otherwise, using the CLI tool directly in terminal is a good way to use Quarto.

**Reply by mcanouil** [reply]
I know, but in the meantime I prefer to do the wrapping myself when calling quarto from R.

**Comment by cderv** [comment, marked as answer]
By "in R", are you talking about **quarto** R package ? 

We still need to add support for all new arguments in the function:  https://github.com/quarto-dev/quarto-r

**Reply by mcanouil** [reply]
By the way, I did not released at first, but a "simple" update of the `quarto` R package is not going to work, at least for everything but book/website projects, see #2171.
